doozerlib/cli/__main__.py

```diff
@@ -1126,6 +1125,15 @@ def images_mirror_streams(runtime, streams, dry_run):
             brew_image = config.image
             brew_pullspec = runtime.resolve_brew_image_url(brew_image)
             cmd = f'oc image mirror {brew_pullspec} {upstream_dest}'
+            registry_config_file = ''
+            if runtime.registry_config is not None:
+                for f in os.listdir(runtime.registry_config):
```

Comment:
I would rather just assume the file is named `config.json`. and check that it exists. also, make sure to expand relative paths, since the CWD changes when we're actually using this. this ought to work:

`DOCKER_CONFIG=./auth -g openshift-3.11 -i configmap-reload images:push --to-defaults`